src/main/java/seedu/address/logic/commands/Command.java

```diff
@@ -19,4 +20,8 @@ public abstract class Command {
 
     public abstract String getCommandWord();
 
+    public String undo(Model model, CommandHistory pastCommands) {
```

Comment:
Shouldn't this be an abstract method instead? Also, I think it would be better if there is a JavaDoc comment indicating the possibility of this method returning `null`.